Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/MarkerDetection/ArUco/HoloLens/ArUcoMarkerAnchor.cs

```diff
@@ -111,6 +111,21 @@ private void OnDestroy()
             markerDetector.MarkersUpdated -= MarkerDetector_MarkersUpdated;
 
         }
+#else
+        public void HandleCommand(SocketEndpoint endpoint, string command, BinaryReader reader, int remainingDataSize)
+        {
+            throw new System.NotImplementedException();
```

Comment:
nit: There should be a better exception: PlatformNotSupported I believe, or something of the sort.